README.md

```diff
@@ -110,58 +115,51 @@ The **reading order** for a **reading-ordered focus navigation scope** is determ
 
 <img src="images/example-1-graph.png" width="400" />
 
-Follows the order-modified document order, unless the order property has been used to change the order of items.
-
 **Forward navigation**
 
-Current is null or outside the scope
+Find first focusable element in scope:
```

Comment:
We don't have steps for finding First() and Last() in this explainer. The spec is not being precise on this.
See 6.6.5 Sequential focus navigation, step 8.

> [...] the user agent should transfer focus to its own controls appropriately [...]

Reply:
I'm not sure step 8 is what you're looking for. That's about what happens when you've tabbed past the last thing on the page, and the focus moves to the browser (e.g. the Omnibox), which is "its own controls".

I think you can say (and it's precise according to the spec) "Start at the first element in the **tabindex ordered focus navigation scope**". You've modified that correct above to include reading order containers.

Reply:
Oh, I understood that as how to find first/last within a scope. Updated examples.